doc/Gabc.tex

```diff
@@ -283,6 +282,15 @@ \subsection{Simple Slurs}
 Use \texttt{uslur} instead of \texttt{oslur} (with either form) to create an
 under-the-staff slur.
 
+\subsection{Horizontal episema placement for very high and low notes}
+
+Gregorio places horizontal episemus under c and above k closer to the notes when no ledger line is present.
```

Comment:
Right, I'm having hard time getting used to English taking the native plural... French foreign words (Latin and Greek are considered foreign) take French plural with s (so episemus, puctums, virgas, etc.), fixing